flink-clients/src/test/java/org/apache/flink/client/program/rest/RestClusterClientTest.java

```diff
@@ -677,6 +682,37 @@ public void testGetAccumulators() throws Exception {
 		}
 	}
 
+	/**
+	 * Tests that command line options override the configuration settings.
+	 */
+	@Test
+	public void testRESTManualConfigurationOverride() throws Exception {
+		final String localhost = "localhost";
```

Comment:
Maybe `configuredHostname`, to be consistent with `manualHostname`. `localhost` is an arbitrary value, and should not be the name of the variable.